tensorboard/webapp/notification_center/_views/notification_center_container.ts

```diff
@@ -14,21 +14,38 @@ limitations under the License.
 ==============================================================================*/
 import {Component} from '@angular/core';
 import {Notification} from '../_redux/notification_center_types';
+import {notificationNotes} from './notification_notes';
+
+type StringDict = {
+  [key: string]: string;
+};
+const iconMap: StringDict = {
+  "What's New": 'info_outline_24px',
+};
 
 @Component({
   selector: 'notification-center',
   template: `
     <notification-center-component
       [notifications]="notifications"
+      [dateStringList]="dateStringList"
+      [iconList]="iconList"
     ></notification-center-component>
   `,
 })
 export class NotificationCenterContainer {
-  notifications: Notification[] = [
-    {
-      date: 1579766400000,
-      title: '2.4 release',
-      content: '<li>update 1</li><li>update 2</li>',
-    },
-  ];
+  notifications: Notification[] = notificationNotes;
```

Comment:
when you do `this.store.select(GetMeAllTheNotifications)`, the return type is `Observable<Notification[]>`.

So, in order to simulate there being a reducer, what you can do is, 
(using rxjs.from)
`from(notificationsNotes)`

Reply:
More completely, 

```ts
interface FeNotification extends Notification {
  icon: string;
}

export class NotificationCenterContainer {
readonly notificationNotes$ = from(notificationsNotes).pipe(map(notifications) => {
  return notifications.map(<your convertNotificationToFeNotification fn>)
}));
}
```